<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSystemToolTip</class>
 <widget class="QWidget" name="WidgetSystemToolTip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>419</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>419</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
  	border: 1px solid #8f8f91;
  	border-radius:6px;
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="2" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="labelSuccess">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgba(0,57,200,1);
</string>
       </property>
       <property name="text">
        <string>修复完成</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1" colspan="3">
      <widget class="QLabel" name="labelIsWifiConnect">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgba(138,143,160,1);</string>
       </property>
       <property name="text">
        <string>确认当前设备是否是无线连接</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="3" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btnConfirm">
       <property name="minimumSize">
        <size>
         <width>88</width>
         <height>44</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>8000</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">  background-color: #0047BB;
  color: rgb(255, 255, 255);
  border:none; 
  padding-left:48px;
padding-right:48px;</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="3" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="labelHelp">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: #C7C7C7;</string>
       </property>
       <property name="text">
        <string>请重试或联系FAE解决</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>54</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="11" column="1" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btnWifiCancel">
       <property name="minimumSize">
        <size>
         <width>88</width>
         <height>44</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>8008</width>
         <height>44</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color:#FAFDFF;
border: 1px solid #DFE3E5;
border-radius: 4px;
color: #2D3440;
padding-left: 10px;
padding-right: 10px;</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="labelMessage">
       <property name="styleSheet">
        <string notr="true">border:none;
color: #C7C7C7;
</string>
       </property>
       <property name="text">
        <string>请重启下控制柜</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" alignment="Qt::AlignRight">
      <widget class="QLabel" name="labelFailedIcon">
       <property name="minimumSize">
        <size>
         <width>66</width>
         <height>66</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>66</width>
         <height>66</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	image: url(:/image/images2/Icon_doingFailed_big.png);
	border: none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QLabel" name="labelFailed">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgba(206,73,73,1);</string>
       </property>
       <property name="text">
        <string>修复失败</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1" alignment="Qt::AlignRight">
      <widget class="QLabel" name="labelWarningIcon">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	image: url(:/image/images2/icon_warning.png);
	border: none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="labelMacAddress">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgba(138,143,160,1);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="3" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btnWifiConfirm">
       <property name="minimumSize">
        <size>
         <width>88</width>
         <height>44</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>8008</width>
         <height>44</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">  background-color: #0047BB;
  color: rgb(255, 255, 255);
  border:none;
  padding-left:24px;
padding-right:24px;</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="3" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="labelWifiIPRecoverSuccess">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color: rgba(138,143,160,1);</string>
       </property>
       <property name="text">
        <string>设置完成请重启控制柜后
检查网关和IP是否可用</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLabel" name="labelRepairing">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
</string>
       </property>
       <property name="text">
        <string>修复中...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1" alignment="Qt::AlignRight">
      <widget class="QLabel" name="labelSuccessIcon">
       <property name="minimumSize">
        <size>
         <width>66</width>
         <height>66</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>66</width>
         <height>66</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	image: url(:/image/images2/Icon_doingSuccessed_big.png);
	border: none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="14" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
